core/tests/test_utils.py

```diff
@@ -3306,7 +3303,7 @@ def save_new_valid_exploration(
             init_state, interaction_id, content_id_generator)
 
         exploration.objective = objective
-        exploration.correctness_feedback_enabled = correctness_feedback_enabled
+        exploration.correctness_feedback_enabled = feconf.DEFAULT_CORRECTNESS_FEEDBACK_ENABLED # pylint: disable=line-too-long
```

Comment:
This should not be necessary at all. Ditto below.

Reply:
removed it.